### ISSUE TITLE
Workload of query on conductor

### ISSUE THREAD

**Issue by fanwar12** [danielgerlag/conductor#17]
Love your work, We are planning to develop something in the like of 5k different workflows, just like to if conductor will support this, keeping in mind that 5K workflow might be running with 50 instance of each workflow at any given time, so 5kx50=250000 instances ?

Moreover, just like to know if conductor support Multi-node clusters.

**Comment by danielgerlag** [comment]
yes, if you configure a redis backplane, then you can run a fleet of nodes as a cluster
